doc: samples: bluetooth: peripheral_hids_keyboard: Fix testing steps (#21925)
nrfconnect/sdk-nrf: samples/bluetooth/peripheral_hids_keyboard/README.rst

         The first one has the value ``02000F0000000000``, the second has the value ``0200000000000000``.
         These values correspond to press and release of character "l" with the Shift key pressed.
      #. In the app, select the HID Report (which has UUID ``0x2A4D`` and the properties ``Read``, ``WriteWithoutResponse``, and ``Write``).

>          Enter any number that has bit 1 set to ``1`` (e.g. ``02``) in the text box and click the :guilabel:`tick mark` button.

Reviewer: Consider turning on/off only Caps Lock LED here (as other output report bits are not used by the sample):
```suggestion
         Enter ``02`` in the text box and click the :guilabel:`tick mark` button.
```
Also in other places

Author: I actually mention that other bits are unused. I think the current version gives the user better understanding of the topic and we still suggest to enter 02. The whole issue comes from the fact that it was not clear for users what other values do and why,

         This sets the Output Report to the written value, in which setting bit 1 to ``1`` simulates turning Caps Lock ON.

>          Other bits are responsible for controlling other Output Report fields, which are not reflected in the sample's UI like e.g. Num Lock.

Reviewer 2: ```suggestion
         Other bits are responsible for controlling other Output Report fields that are not reflected in the sample's UI, for example Num Lock.
```

         Observe that **LED 3** is lit.
      #. Select the same HID Report again.

>          Enter any number that has bit 1 set to ``0`` (e.g. ``00``) in the text box and click :guilabel:`Write`.

Reviewer 2: ```suggestion
         Enter any number that has bit 1 set to ``0`` (for example ``00``) in the text box and click :guilabel:`Write`.
```

         The first one has the value ``02000F0000000000``, the second has the value ``0200000000000000``.
         These values correspond to press and release of character "l" with the Shift key pressed.
      #. In the app, select the HID Report (which has UUID ``0x2A4D`` and the properties ``Read``, ``WriteWithoutResponse``, and ``Write``).

>          Enter any number that has bit 1 set to ``1`` (e.g. ``02``) in the text box and click the :guilabel:`tick mark` button.

Reviewer 2: ```suggestion
         Enter any number that has bit 1 set to ``1`` (for example ``02``) in the text box and click the :guilabel:`tick mark` button.
```

         This sets the Output Report to the written value, in which setting bit 1 to ``1`` simulates turning Caps Lock ON.

>          Other bits are responsible for controlling other Output Report fields, which are not reflected in the sample's UI like e.g. Num Lock.

Reviewer 2: ```suggestion
         Other bits are responsible for controlling other Output Report fields that are not reflected in the sample's UI, for example Num Lock.
```

         Observe that **LED 2** is lit.
      #. Select the same HID Report again.

>          Enter any number that has bit 1 set to ``0`` (e.g. ``00``) in the text box and click :guilabel:`Write`.

Reviewer 2: ```suggestion
         Enter any number that has bit 1 set to ``0`` (for example ``00``) in the text box and click :guilabel:`Write`.
```
